fuel-tests/tests/tx/predicates.rs

```diff
@@ -0,0 +1,75 @@
+// Tests related to the predicate execution feature
+
+use crate::helpers::TestSetupBuilder;
+use fuel_crypto::Hasher;
+use fuel_tx::{Input, Output, TransactionBuilder};
+use fuel_vm::{consts::REG_ONE, prelude::Opcode};
+use rand::{rngs::StdRng, Rng, SeedableRng};
+
+#[tokio::test]
+async fn transaction_with_predicates_is_rejected_when_feature_disabled() {
+    let mut rng = StdRng::seed_from_u64(2322);
+
+    // setup tx with a predicate input
+    let asset_id = rng.gen();
+    let predicate_tx = TransactionBuilder::script(Default::default(), Default::default())
+        .add_input(Input::coin_predicate(
```

Comment:
This should break the validation for the next bump

https://github.com/FuelLabs/fuel-tx/blob/c9704af3ba826beb559623912fbd735a2303b32e/src/transaction/types/input.rs#L255-L261

We should set the owner as the code root of the validator

Reply:
thanks for the heads up, will deal with it in the next pr